metabrainz/templates/index/privacy.html

```diff
@@ -7,12 +7,144 @@ <h1 class="page-title">{{ _('Privacy Policy') }}</h1>
+    ask is that you: choose a unique username and password, use a web browser that accepts "session" cookies, and provide a verified 
+    email address.
+  </p>
+  <p>
+    If you do not create an account (or are not logged in) you will not have access to the above listed features, however, you will 
+    still have full access to browse and examine the database.
+  </p>
+
+  <h2>{{ _('Applicable to all users') }}</h2>
+
+  <h3>{{ _('Cookies') }}</h3>
+  <p>
+    We use two cookies that contain sensitive information:
+  </p>
+
+  <ol>
+    <li>
+      <b>remember_login</b>: If you choose the "Log in permanently" option when logging in to one of our sites; the cookie will be used to 
+      remember your login details.
+    </li>
+    <li>
+      <b>musicbrainz_server_session</b>: If you are logged into one of our sites, your current session ID is stored in this cookie.
+    </li>
+  </ol>
+
+  <p>
+    We may store boolean dynamic preference information in other cookies, but these do not contain any sensitive information.
+  </p>
+
+  <h3>{{ _('Web and FTP access logs') }}</h3>
+
+  <p>
+    In a practice similar to other web sites we keep logs of all web requests made against our servers. These logs include: your IP 
+    address, your browser's "User-Agent" string, and which page you requested. Aggregate information about web and FTP traffic is 
+    made available to the public via our site usage pages.
+  </p>
+
+  <h3>{{ _('Third-Party content') }}</h3>
+  <p>
+    Our project's web pages may load some third-party content. Some, but possibly not all sites are listed below:
+  </p>
+
+  <ol>
+    <li>
+      Cover art images: These are provided by the various cover art sites that have given MusicBrainz permission to do so. This means 
+      that the relevant site will know (if they want to) your IP address, User-Agent string, etc., and which MusicBrainz web page you 
+      were visiting (the one which included the cover art image).
+    </li>
+    <li>
+      Google Analytics: We use Google Analytics in our web pages to get a picture of who visits MusicBrainz. From these analytics we 
+      can set targets for supported browser versions, screen sizes and other useful information that we take into consideration when 
+      developing or improving features on our site. If you object to the use of Google Analytics, please consider blocking it in your 
+      browser.
+    </li>
+    <li>
+      AcoustID: Our fingerprints tab loads fingerprint information from acoustid.org.
+    </li>
+    <li>
+      reCAPTCHA: Our account creation page loads third party content in order to prevent spammers from creating accounts.
+    </li>
+  </ol>
+
+  <p>
+    The above information assumes that you are using "normal" web browser settings, whereby images are always loaded and HTTP referrer 
+    information is always sent.
+  </p>
```

Comment:
[The current version on MB.o](https://wiki.musicbrainz.org/About/Privacy_Policy?oldid=63692#Third-party_content) has this section italicised to make it appear a bit more parenthetical/as a sidenote/gotcha. Maybe copy that formatting here too?

Reply:
@mayhem I think maybe you missed this?